requirements.txt

```diff
@@ -1,12 +1,44 @@
-Flask==0.10.1
-Jinja2==2.7.3
-MarkupSafe==0.23
-Werkzeug==0.9.6
-gnureadline==6.3.3
-itsdangerous==0.24
-rauth==0.7.0
-requests==2.3.0
-wsgiref==0.1.2
-
-gunicorn==18.0
-Flask-SSLify==0.1.4
+aniso8601==7.0.0
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2021-28363   
**Check Name:** python-urllib3: HTTPS proxy host name not validated when using default SSLContext   
**Severity:** MEDIUM   
**Fixed Version:** 1.26.4   
**Description:** The urllib3 library 1.26.x before 1.26.4 for Python omits SSL certificate validation in some cases involving HTTPS to HTTPS proxies. The initial connection to the HTTPS proxy (if an SSLContext isn't given via proxy_config) doesn't verify the hostname of the certificate. This means certificates for different servers that still validate properly with the default urllib3 SSLContext will be silently accepted.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2021-28363